src/domain/httpsactuatorserver.py

```diff
@@ -0,0 +1,95 @@
+"""HTTPS Actuator server module"""
+
+import json
+import logging
+import re
+import ssl
+import threading
+from http import HTTPStatus
+from http.server import BaseHTTPRequestHandler, HTTPServer
+
+from actuator import Actuator
+
+logger = logging.getLogger(__name__)
+
+
+class HTTPActuatorRequestHandler(BaseHTTPRequestHandler):
+    def do_GET(self):
+        if re.search("/api/v1/actuators/*", self.path):
+            logger.info(
+                "Connected remote actuator from %s:%s", self.client_address[0], self.client_address[1]
+            )
+
+            record_id = self.path.split("/")[-1]
+
```

Comment:
Not the best, as it will match also call like `/api/v1/actuators/this/is/a/random/call`
I assume `record_id = 'this/is/a/random/call'` is not the desired behaviour

Reply:
Again flask/fastapi provides a very nice way to be very stringent with routes definition

Reply:
I agree with you. When we define a format for the actuator ID we can perform a more precise check, or if we think it's appropriate, we can switch to Flask/FastAPI.

Reply:
Implemented using FastAPI